lib/inspec/cli.rb

```diff
@@ -321,6 +321,9 @@ def detect
     desc: "A space-delimited list of local folders containing profiles whose libraries and resources will be loaded into the new shell"
   option :distinct_exit, type: :boolean, default: true,
     desc: "Exit with code 100 if any tests fail, and 101 if any are skipped but none failed (default).  If disabled, exit 0 on skips and 1 for failures."
+  option :command_timeout, type: :numeric, default: 60,
```

Comment:
I don't think this is needed. The important thing for shell is that users can paste the code that they would be running in `exec`. 
I'll remove this and test that a timeout set at the `command` resource still works in shell.